app/models/dba/partition_all.rb

```diff
@@ -0,0 +1,52 @@
+class DBA::PartitionAll
+  TABLES = %w[
```

Comment:
If we're going to do all of the tables, it might be nicer to grab them like this:
```
HmisCsvImporter::Loader::Loader.loadable_files.values.map(&:table_name)
HmisCsvImporter::Importer::Importer.importable_files.values.map(&:table_name)
```
That said, things like `hmis_csv_2022_affiliations` and `hmis_2022_users` and a handful of others will never be very big.

Reply:
I had started to ask how to find the tables in code but didn't want to bug you. I can make that change, but *not* partitioning anything we can avoid would be good to avoid another 70+ partitions.

Reply:
Tables with the following in their names can be ignored for partitioning.
```
exports
organizations
projects
funders
inventories
project_cocs
affiliations
users
```